mtp_bank_admin/apps/bank_admin/refund.py

```diff
@@ -12,13 +12,29 @@
 
 def generate_refund_file(request):
     client = api_client.get_connection(request)
-    refund_transactions = client.bank_admin.transactions.get(status='refund_pending')
+    response = client.bank_admin.transactions.get(
+        status='refund_pending',
+        limit=settings.REQUEST_PAGE_SIZE
+    )
+    transactions_to_refund = response.get('results', [])
+    total_count = response.get('count', 0)
+
+    num_reqs = 1
```

Comment:
Given you're using `settings.REQUEST_PAGE_SIZE`, it might be clearer if this was `page_num`?